### PR TITLE
CIP-0025 | Update "image" Spec Based on Feedback

### PR DESCRIPTION
**Pull request by thaddeusdiamond**
Practical usage has led to a proliferation of URI mechanisms for the "image" field in CIP-0025 metadata.  (See
https://twitter.com/pool_pm/status/1575219209491337216?s=20&t=KZKbGeCKIUnPyTZ0fzPyhQ for one thread on the topic).  This change would be a simple clarification in one sub-bullet for newly introduced community members looking to get familiar with the standard.

Since IPFS and Arweave are the most common methods of data storage for NFT files, we explicitly call them out and provide more resources. Note that this is particularly important as certain free storage providers like NFT.Storage only return the version 1 CID in their file upload calls and lead to scheme being commonly omitted in the URI.

### CODE REVIEW

**Comment by thaddeusdiamond** [comment]
cc @SmaugPool @KtorZ @alessandrokonrad

**Comment by KtorZ** [comment]
This sounds like a reasonable precision. I am not quite sure what you mean with the following sentence:

> not merely the common identifier as NFT viewers may not be able to locate the file

Would it be worth clarifying with a few explicit example perhaps? Good examples usually help clearing out ambiguity.

**Review by rphair — COMMENTED** [review]
This seems like a good clarification to have & I would have appreciated it at my own level, especially with the link to OpenSea.  Happy to approve pending the detail suggested in https://github.com/cardano-foundation/CIPs/pull/341#issuecomment-1264267886.

**Comment by thaddeusdiamond** [comment]
I meant "content" not "common" woops (https://github.com/multiformats/cid).  Perils of writing late at night.  Including more examples and resubmitting.  @rphair and @KtorZ let me know feedback on this second commit.  I am attaching a photo of how it shows up in stackedit.io

Screen Shot 2022-10-02 at 2 49 42 PM

File: CIP-0025/README.md [hunk 1]
```diff
@@ -78,7 +78,20 @@ The structure allows for multiple token mints, also with different policies, in
...
...S4zM0g3MzBWMTg2LjkxaDQ4LjU2YzM2LjY4LDAsNTktMTkuMTQsNTktNjAuNzlTODE1LjI2LDY1LjMzLDc3OC41OCw2NS4zM1ptMzUuOCw2Mi41NmMtLjE4LDI2LjU5LTEyLjc2LDM5LjctMzUuOCwzOS43SDc1Mi43MVY4NC42NWgyNS44N2MyMywwLDM1LjgsMTMuMTIsMzUuOCwzOS43WiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik04OTcsNjUuMzMsODQ5LjY2LDE4Ni45MWgyMy4zOWw5Ljc1LTI2LjQxaDU1LjY1bDkuOTIsMjYuNDFoMjQuNDZMOTI1LjUxLDY1LjMzWm0tNy4wOSw3Nkw5MDIuMywxMDhjMi40OC02LjU1LDYuOTEtMjAuNzMsNy44LTIzLjIxSDkxMWMuODgsMi40OCw1LjMyLDE2LjgzLDcuOCwyMy4yMWwxMi40LDMzLjMyWiIgZmlsbD0iIzAwMzNhZCIvPjxwYXRoIGQ9Ik0xMDc2LjU0LDEzNy42NGMwLDIuMy4zNSw4LjUuMzUsOS41N2wtLjg4LjM1Yy0uNTMtLjctNC4wOC02LjkxLTYtOS43NGwtNTIuODEtNzIuNDlIOTk1LjU1VjE4Ni45MWgyMS44VjExNC4wN2MwLTMtLjM2LTguODYtLjM2LTkuNzVsLjg5LS41M2MuNTMuNzEsMy4xOSw1Ljg1LDYsOS45M2w1My4xNyw3My4xOWgyMS4yNlY2NS4zM2gtMjEuNzl2NzIuMzFaIiBmaWxsPSIjMDAzM2FkIi8+PHBhdGggZD0iTTExODguNjgsNjMuMjFjLTM4LjEsMC02MS4xNCwyMC4yLTYxLjE0LDYyLjkxczIzLDYyLjkxLDYxLjE0LDYyLjkxLDYxLjMyLTIwLjIsNjEuMzItNjIuOTFTMTIyNi43OCw2My4yMSwxMTg4LjY4LDYzLjIxWm0zOC4xLDY0Ljg2YzAsMjcuODItMTMuMjksNDEuODItMzguMSw0MS44Mi0yNC42MywwLTM3LjkyLTE0LTM3LjkyLTQxLjgydi0zLjcyYzAtMjcuODMsMTMuNDctNDIsMzcuOTItNDIsMjQuNjQsMCwzOC4xLDE0LjE3LDM4LjEsNDJaIiBmaWxsPSIjMDAzM2FkIi8+PC9nPjwvZz48L3N2Zz4=`
```
Comment (rphair) [inline CIP-0025/README.md:93]:
```suggestion
```
The section of included code doesn't say anything additional about the property usage beyond repeating what you already quoted (`data:<mime_type>;base64`).  It takes a disproportionate part of the screen & the document itself, especially on small screen devices (we do have CIP readers on those, as we found out with another CIP).

What about another link, more user friendly than the RFC, that goes into more detail about broader usage of image types, compatibility, and constructing the base64 data?  https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs

**Review by rphair — COMMENTED** [review]
Additional detail is very helpful... ready to approve given the reservation below 😎

**Comment by thaddeusdiamond** [comment]
Good callouts.  Rebased the branch on top of @rphair's whitespace commit.  Feel free to merge whenever this meets the review needs.

**Review by rphair — APPROVED** [review]
I'm happy with it & believe it now fully satisfies https://github.com/cardano-foundation/CIPs/pull/341#issuecomment-1264267886 😎

**Review by KtorZ — APPROVED** [review]

**Pull request merged**